tensilelite/Tensile/Source/lib/include/Tensile/PropertyMatching.hpp

```diff
@@ -975,6 +975,19 @@ namespace Tensile
                 bool      Debug = T_Debug;
                 std::cout << std::setprecision(2) << std::fixed;
 
+                Key key = key_orig;
```

Comment:
Do we need to keep key_orig?

Reply:
We don't want original key to be modified, in case there are more hierarchical search in the future, like equality -> gridbased -> stream-k -> freesize

Reply:
It is not a good idea to change Key const & to Key at line 964.
We should be able to use copy constructor here without changing the type at line 964.

Reply:
done